README.md

```diff
@@ -39,7 +39,7 @@ import (
 
 func main() {
 	r := gin.Default()
-	r.Use(gzip.Gzip(gzip.DefaultCompression))
+	r.Use(gzip.Gzip(gzip.DefaultCompression, 0))
```

Comment:
I'm not part of the project but can you add this by using options pattern so it doesn't break all the existing code out there.